### ISSUE TITLE
(Suggestion) Toggle Voice Input

### ISSUE THREAD

**Issue by yndrelbosch** [YihaoPeng/DragonbornSpeaksNaturally#5]
It would be nice to have a function where you can toggle the listening function. Use case would be, you have someone talking to you in Real, you toggle the input, and respond. Otherwise, if you respond sometimes DSN thinks you are talking to it, and will execute a command.

**Comment by SwimmingTiger** [comment]
This feature has been completed in the following commit: https://github.com/YihaoPeng/DragonbornSpeaksNaturally/commit/336761c0076eb103b42f5610a44898e1fdd46be6
Feedback on this work is welcome.

**Comment by SwimmingTiger** [comment]
Because there is no further feedback, the issue is temporarily closed.

**Comment by Kuiriel** [comment]
I found the existing Windows Speech Recognition features to be adequate previously - I just needed it to stop turning ON the listening features automatically even after I turned them off. Even when I made the command complex and sound completely different when turning on vs off.

Having this bound to a key press would be a bit of a saver, especially for immersion when recording/streaming.

Trying to use Voice2JSON to see if this solves it, just need to get it to work in game, ha...
